test/cmdlineTests/viair_msize_without_optimizer/output

```diff
@@ -84,13 +82,8 @@ sub_0: assembly {
       swap1
       jump	// out
     tag_3:
-      0x00
-      dup1
-      revert
     tag_4:
-      0x00
-      dup1
-      revert
+      revert(0x00, 0x00)
```

Comment:
Here it may be a problem that we get two tags at the same position - need to check why that happens - the block deduplicator should merge the tags instead